bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,325 @@
+            'score': 0,
+        }
+
+        for emoji in self.emojis_val:
+            await message.add_reaction(emoji)  # add each emoji
+
+        logger.info(f"{message.author} added the word {word!r}")
+
+        await ctx.send(f"{word!r} added successfully!\n{random.choice(self.added_messages)}")
+
+    @Cog.listener()
+    async def on_reaction_add(self, reaction: discord.Reaction, user: discord.User) -> None:
+        """Ensures that each user adds one and only one reaction."""
+        try:
+            if reaction.emoji in self.emojis_val and reaction.message.id in self.messages:
+                msg: Message = reaction.message
+                # create a custom counter
+                reaction_counter = defaultdict(int)
+                for r in msg.reactions:
+                    async for user in r.users():
+                        reaction_counter[user] += 1
+
+                if reaction_counter[user] > 1 and user != self.bot.user:
+                    await user.send("Sorry, you have already added a reaction, please remove your reaction and try "
+                                    "again")
+                    return await reaction.remove(user)  # removes user's reaction
+        except RuntimeError:  # The dictionary was changed in between
+            pass
+
+    @spook_name_rate.command(name='delete')
+    async def delete_word(self, ctx: Context) -> None:
+        """Delete the user's word."""
+        if self.poll:
+            await ctx.send("You can't delete your word since the poll has already started!")
+            return
+        for message_id in self.messages:
+            data = self.messages[message_id]
+            if ctx.author == data['author']:
+                del self.messages[message_id]
+                await ctx.send(f'Message deleted successfully ({data["word"]!r})!')
+                return
+
+        await ctx.send("But you don't have an entry... :eyes: Type `spooknamerate add your entry`")
+
+    @tasks.loop(hours=24.0)
+    async def announce_word(self) -> None:
+        """Announces the name needed to spookify every 24 hours and the winner of the previous game."""
+        if not datetime.utcnow().month == Month.OCTOBER:
+            return
+
+        channel = await self.get_channel()
+
+        if self.first_time:
+            await channel.send("Okkey... Welcome to **Spook Name Rate**! It's a relatively simple game.\n"
+                               "Everyday, a random name will be sent in `#seasonalbot-commands` "
+                               "and you need to try and spookify it!\nRegister your word using "
+                               f"`{self.bot.command_prefix}spooknamerate add spookified_name`")
+
+            self.first_time = False
+
+        else:  # otherwise,
+            if self.messages:
+                await channel.send(embed=await self.get_responses_list(final=True))
+                self.poll = True
+                await asyncio.sleep(2 * 60 * 60)
```

Comment:
2 hours? a comment or variable for these kinds of magic numbers are very useful

Reply:
Wait. I had added a [comment](https://github.com/python-discord/seasonalbot/pull/509/commits/0fd87708d416570747180a3215ea1c56ed3c18b7#diff-cbfaa2fbf8a2b4f73ab3f511bbe9c527d71b8f4cca821cfc737593cabea2a107L208), but I removed it thinking it was redundant since I had already written it out (2 * 60 * 60 instead of 7200 directly). Silly me 😆.